--- wscript-orig	2011-02-24 04:34:32.340295382 -0500
+++ wscript	2011-02-24 04:34:42.627298675 -0500
@@ -208,7 +208,7 @@
 		conf.fatal('Compiling Semantik requires kde4 -devel')
 
 	conf.env.OCALINKFLAGS_OPT_PP = 'str.cmxa'
-	conf.env.LINKFLAGS_CAML = '-lasmrun -lstr -lnums'.split()
+	conf.env.LINKFLAGS_CAML = '-lasmrun -lnums'.split()
 	conf.env.LIBPATH_CAML = conf.env.OCAMLLIB
 	conf.env.LIB_CAML = "m".split()
 	conf.env.OCAMLFLAGS_TWT = '-pp default/util/ocamltwt'.split() #-unsafe -noassert -inline 10'
