--- semantik-1.0.2/src/base/sembind_py.cpp.orig	2018-02-28 22:06:10.430824888 +0000
+++ semantik-1.0.2/src/base/sembind_py.cpp	2018-02-28 22:06:24.841319204 +0000
@@ -33,7 +33,7 @@
 	}
 
 	Py_ssize_t l_iSize = 0;
-	char * l_iChars = PyUnicode_AsUTF8AndSize(i_o, &l_iSize);
+	const char * l_iChars = PyUnicode_AsUTF8AndSize(i_o, &l_iSize);
 
 	QString l_s = QString::fromUtf8(l_iChars, l_iSize);
 	QChar * data = l_s.data();
